docs/src/content/docs/es/resources/community-content.mdx

```diff
@@ -139,5 +144,11 @@ import YouTubeGrid from '~/components/youtube-grid.astro';
 			description:
 				'Configura Starlight como un proyecto de subdirectorio dentro de un sitio web de Next.js',
 		},
+		{
```

Comment:
I think this is missing this [video](https://github.com/withastro/starlight/blob/main/docs/src/content/docs/resources/community-content.mdx?plain=1#L143-L148).